### PR TITLE
Fixing grammar across the site

### PR DESCRIPTION
**Pull request by mfwmyfacewhen**
Fix grammar across the site

### CODE REVIEW

**Comment by qua3k** [comment]
Not all of these comma errors are comma splices.

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h5 class="mt-2">“I have nothing to hide. Why should I care about my privacy?
     <div class="col-lg-8 col-12 text-end">
       <h2 class="mt-4">What should I do?</h2>
       <h5 class="mt-2">First, you need to make a plan.</h5>
-      <p class="mt-2">Trying to protect <strong>all</strong> your data from <strong>everyone</strong> all the time is impractical, expensive, and exhausting. But, don't worry! Security is a process, and by thinking ahead you can put together a plan that's right for <strong>you</strong>. Security isn't just about the tools you use or the software you download. Rather, it begins with understanding the unique threats you face, and how you can counter them.</p>
+      <p class="mt-2">Trying to protect <strong>all</strong> your data from <strong>everyone</strong> all the time is impractical, expensive, and exhausting. But don't worry! Security is a process, and by thinking ahead you can put together a plan that's right for <strong>you</strong>. Security isn't just about the tools you use or the software you download. Rather it begins with understanding the unique threats you face, and how you can counter them.</p>
```
Comment (qua3k) [inline index.html:37]: Rather's comma shouldn't be removed as it starts a sentence

Comment (mfwmyfacewhen) [reply index.html:37]: k fixing

File: _data/operating-systems/android/2_calyxos.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ description: |
   Currently, CalyxOS [supports](https://calyxos.org/docs/guide/device-support/) Google Pixel phones and the Xiaomi Mi A2. For legacy devices, CalyxOS offers "extended support" for much longer than GrapheneOS, making it a good choice once GrapheneOS has dropped support.
 
   #### Notes
-  CalyxOS's "extended support" does not have full security patches due to the original equipment manufacturer (OEM) discontinuing support, therefore they cannot be considered completely secure.
+  CalyxOS's "extended support" does not have full security patches due to the original equipment manufacturer (OEM) discontinuing support; therefore they cannot be considered completely secure.
```
Comment (qua3k) [inline _data/operating-systems/android/2_calyxos.yml:14]: “Therefore” should be followed by a comma.

**Review by qua3k — CHANGES_REQUESTED** [review]

**Comment by qua3k** [comment]
There are a lot more issues that can be found simply by grepping for commas. This PR only fixes some of them.

**Comment by mfwmyfacewhen** [comment]
> There are a lot more issues that can be found simply by grepping for commas. This PR only fixes some of them.

I think we ought to merge this as is and then I'll do a pr for each section separately.

**Pull request merged**